### PR TITLE
update badge link

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
@codeclimate/review Updates badge link

### CODE REVIEW

**Comment by pbrisbin** [comment]
Looks like this is hitting the same error @mrb was...

**Comment by wfleming** [comment]
`gem install io-console` in the Dockerfile was his fix. I still think a root cause must be something with the alpine ruby image.

**Comment by ABaldwinHunter** [comment]
@pbrisbin @wfleming For this issue, I can try to do `gem install io-console`.

Or, do we want to use our own copy of `alpine-ruby` stored somewhere?

**Comment by pbrisbin** [comment]
I think it makes sense to move to our alpine-ruby base image if that's what we do for all our other ruby-based engines. I don't feel strongly though, and moving to some other defacto ruby base image is also fine with me.

**Comment by ABaldwinHunter** [comment]
@codeclimate/review I updated the `Dockerfile` to use `codeclimate/alpine-ruby:b36` instead of `alpine:edge` and the tests pass. 

I'm also okay waiting on this change.

Eventually, it might be preferable to use `ruby:2.2.2` but that will involve a more in depth change to the way we run `app` user.

**Comment by pbrisbin** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
-FROM alpine:edge
+FROM codeclimate/alpine-ruby:b36
+
+WORKDIR /usr/src/app
```
Comment (GordonDiggs) [inline Dockerfile]: These lines are duplicates

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,8 @@
-FROM alpine:edge
+FROM codeclimate/alpine-ruby:b36
```
Comment (GordonDiggs) [inline Dockerfile]: Should this be 38?

Comment (ABaldwinHunter) [inline Dockerfile]: I believe we only have `b36` publicly atm https://hub.docker.com/r/codeclimate/alpine-ruby/tags/

```
$ docker pull codeclimate/alpine-ruby:b38
Pulling repository codeclimate/alpine-ruby
FATA[0001] Tag b38 not found in repository codeclimate/alpine-ruby 
```

File: Dockerfile (hunk 1 above)
Comment (ABaldwinHunter) [inline Dockerfile]: thanks!

**Comment by GordonDiggs** [comment]
LGTM

**Comment by ABaldwinHunter** [comment]
@codeclimate/review `codeclimate/alpine-ruby:b38` is now available on Dockerhub (thanks @GordonDiggs!) so I opted to use that image instead. (It's used by `scheduler`). 

Using this new `rubocop`, I successfully ran analysis locally via CLI `codeclimate analyze` and with SOA of `harness`.

**Pull request merged**